pkg/helmt/helmt.go

```diff
@@ -124,6 +125,11 @@ func template(name string, chart string, values []string, namespace string, skip
 	} else {
 		args = append(args, "--output-dir", ".")
 	}
+	if len(ApiVersions) > 0 {
+		for _, apiversion := range ApiVersions {
+			args = append(args, "--api-versions", apiversion)
```

Comment:
are you sure that you can specify api version flag multiple times?

`-a, --api-versions stringArray     Kubernetes api versions used for Capabilities.APIVersions`

Reply:
specifying multiple time works but when I specify as key=value and string seperated by comma it doesn't work

Reply:
these cases don't work
`--api-versions "monitoring.coreos.com/v1","monitoring.coreos.com/v1alpha1"`
`--api-versions 0="monitoring.coreos.com/v1",1="monitoring.coreos.com/v1alpha1"`